test_conformance/vulkan/test_vulkan_platform_device_info.cpp

```diff
@@ -76,7 +114,20 @@ int test_platform_info(cl_device_id deviceID, cl_context _context,
 
     for (i = 0; i < num_platforms; i++)
```

Comment:
I think it would be more appropriate here to get the platform to test from the passed-in device handle rather than to loop over all of the platforms in the system.  Rationale: the CTS tests are testing a single device (and hence a single platform), chosen by the harndess.  The test results should be reported for that single device, and should not be affected by any other (possibly buggy or incomplete) platforms that are installed in the system.

Reply:
Yeah, makes sense, will update.